op-service/sources/types_test.go

```diff
@@ -70,7 +70,8 @@ func TestBlockJSON(t *testing.T) {
 			var block RPCBlock
 			readJsonTestdata(t, "testdata/data/blocks/"+strings.Replace(entry.Name(), "_metadata.json", "_data.json", 1), &block)
 
-			err := block.Verify()
+			l2RpcChecker := NewL1RPCChecker()
```

Comment:
This seems strange, why is the L1 RPC checker being assigned to a variable called l2 rpc checker?